awscli/customizations/wizard/wizards/configure/main.yml

```diff
@@ -0,0 +1,361 @@
+        cache: true
+        paginate: true
+      credential_source_type:
+        type: prompt
+        description: Select your credentials source type
+        choices:
+          - display: Source Profile
+            actual_value: source_profile
+          - display: Environment Variables
+            actual_value: Environment
+          - display: Amazon EC2 Instance Metadata
+            actual_value: Ec2InstanceMetadata
+          - display: Amazon ECS Container Credentials
+            actual_value: EcsContainer
+        condition:
+          - variable: config_type
+            equals: assume_role
+          - variable: add_creds
+            equals: yes
+      source_profile:
+        type: prompt
+        description: Select the source profile
+        choices: existing_profiles
+        condition:
+          - variable: credential_source_type
+            equals: source_profile
+          - variable: config_type
+            equals: assume_role
+          - variable: add_creds
+            equals: yes
+      role_arn:
+        type: prompt
+        description: Select the role you want to assume
+        choices: existing_roles
+        condition:
+          - variable: config_type
+            equals: assume_role
+          - variable: add_creds
+            equals: yes
+      creds_process:
+        type: prompt
+        description: Enter the credential process command
+        condition:
+          - variable: config_type
+            equals: proc_provider
+          - variable: add_creds
+            equals: yes
+  additional_config:
+    shortname: Additional config
+    description: Additional CLI configuration
+    values:
+      add_config:
+        type: prompt
+        description: "Would you like to add additional configuration"
+        choices:
+          - display: Yes
+            actual_value: yes
+          - display: No
+            actual_value: no
+      region_details:
+        type: template
+        value: |
+          Specifies the AWS Region to send requests to for commands requested using this profile.
+
+            You can specify any of the Region codes available for the chosen service as listed in AWS Regions and Endpoints in the Amazon Web Services General Reference.
+
+            "aws_global" enables you to specify the global endpoint for services that support a global endpoint in addition to regional endpoints, such as AWS Security Token Service (AWS STS) and Amazon Simple Storage Service (Amazon S3).
+      region:
+        type: prompt
+        description: "Enter the region name"
+        condition:
+          variable: add_config
+          equals: yes
+        details:
+          value: region_details
+          visible: True
+          description: "Docs"
+      prompt_mode_details:
+        type: template
+        value: |
+          Enables the auto-prompt for the AWS CLI. There are two settings that can be used:
```

Comment:
Similar to what we did with the `iam wizard new-role`, it may be interesting if we labeled the details with `output` and make it `rst` (or any other format) so that:

1. We can right these doc strings in rst or even load it from some other rst file that we have (we may want to consider doing that now depending how much documentation we want embedded into the wizard. The wizard definition is getting quite lengthy already...)
2. Render the values similar to what we do for the man pages that we do in auto-prompt mode

I think that will make the docs panel look cleaner in general.

Reply:
sounds reasonable but I'd prefer to use HTML or Markdown instead and get rid of `rst`